applications/nrf_desktop/README.rst

```diff
@@ -900,6 +908,13 @@ nRF52840 USB Dongle (nrf52840dongle_nrf52840)
         Input data comes from Bluetooth and is retransmitted to USB.
       * |preconfigured_build_types|
 
+nRF52833 USB Dongle (nrf52833dongle_nrf52833) and nRF52820 Dongle (nrf52820dongle_nrf52820)
+      * Both reference designs are meant for the project-specific hardware and are defined in :file:`nrf/boards/arm/nrf52833dongle_nrf52833` and :file:`nrf/boards/arm/nrf52820dongle_nrf52820`, respectively.
```

Comment:
>Both reference designs are meant for the project-specific hardware and are defined in :file:`nrf/boards/arm/nrf52833dongle_nrf52833` and :file:`nrf/boards/arm/nrf52820dongle_nrf52820`, respectively.

This part is not needed

Reply:
> This configuration uses Nordic's nRF52840 dongle defined in Zephyr

If you remove this (line 904) 833 and 840 dongles will go together.

Reply:
done